collect_app/src/main/java/org/odk/collect/android/widgets/RangeDecimalWidget.java

```diff
@@ -40,45 +76,36 @@ public IAnswerData getAnswer() {
     }
 
     @Override
-    protected void setUpActualValueLabel() {
-        String value = actualValue != null
-                ? String.valueOf(actualValue.doubleValue())
-                : "";
-
-        if (currentValue != null) {
-            currentValue.setText(value);
-        }
+    public void setOnLongClickListener(OnLongClickListener l) {
     }
 
     @Override
-    protected void setUpDisplayedValuesForNumberPicker() {
-        displayedValuesForNumberPicker = new String[elementCount + 1];
-
-        if (isRangeIncreasing()) {
-            fillDisplayedValuesWithIncreasingValues();
+    public boolean suppressFlingGesture(MotionEvent e1, MotionEvent e2, float velocityX, float velocityY) {
+        return slider.suppressFlingGesture;
```

Comment:
I like what is happening here! Just the naming and how the widget is able to access the field that is bothering me.